bascule/basculehttp/log.go

```diff
@@ -0,0 +1,34 @@
+package basculehttp
+
+import (
+	"context"
+	"os"
+
+	"github.com/go-kit/kit/log"
+)
+
+var (
+	defaultLogger = log.NewJSONLogger(log.NewSyncWriter(os.Stdout))
+
+	callerKey    interface{} = "caller"
```

Comment:
You don't want to replicate these here.  Just let application code set all this up, e.g. in `gungnir`